pom.xml

```diff
@@ -51,7 +51,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <zookeeper.version>3.5.1-alpha</zookeeper.version>
-    <bookkeeper.version>4.3.4-TWTTR</bookkeeper.version>
+    <bookkeeper.version>4.3.4-TWTTR-SNAPSHOT</bookkeeper.version>
```

Comment:
@arvindkandhare I pushed a version '4.3.5-TWTTR-OSS'. Could you update the version?